edx_global_analytics/analytics/test_fixture.json

```diff
@@ -0,0 +1 @@
stallationstatistics", "fields": {"active_students_amount_day": 2825, "edx_installation": 121, "statistics_level": 1, "active_students_amount_month": 1000, "active_students_amount_week": 1000, "students_per_country": "{\"RU\": 2632, \"CA\": 18543, \"UA\": 2011, \"null\": 1}", "data_created_datetime": "2016-02-22T20:24:26Z", "courses_amount": 113}}, {"pk": 121, "model": "analytics.edxinstallation", "fields": {"latitude": 1.0, "platform_name": "test", "platform_url": "test", "longitude": 1.0, "secret_token": "platform 0"}}, {"pk": 122, "model": "analytics.installationstatistics", "fields": {"active_students_amount_day": 2850, "edx_installation": 122, "statistics_level": 1, "active_students_amount_month": 1000, "active_students_amount_week": 1000, "students_per_country": "{\"RU\": 2632, \"CA\": 18543, \"UA\": 2011, \"null\": 1}", "data_created_datetime": "2016-02-23T20:24:26Z", "courses_amount": 114}}, {"pk": 122, "model": "analytics.edxinstallation", "fields": {"latitude": 1.0, "platform_name": "test", "platform_url": "test", "longitude": 1.0, "secret_token": "platform 0"}}, {"pk": 123, "model": "analytics.installationstatistics", "fields": {"active_students_amount_day": 2875, "edx_installation": 123, "statistics_level": 1, "active_students_amount_month": 1000, "active_students_amount_week": 1000, "students_per_country": "{\"RU\": 2632, \"CA\": 18543, \"UA\": 2011, \"null\": 1}", "data_created_datetime": "2016-02-24T20:24:26Z", "courses_amount": 115}}, {"pk": 123, "model": "analytics.edxinstallation", "fields": {"latitude": 1.0, "platform_name": "test", "platform_url": "test", "longitude": 1.0, "secret_token": "platform 0"}}, {"pk": 124, "model": "analytics.installationstatistics", "fields": {"active_students_amount_day": 2900, "edx_installation": 124, "statistics_level": 1, "active_students_amount_month": 1000, "active_students_amount_week": 1000, "students_per_country": "{\"RU\": 2632, \"CA\": 18543, \"UA\": 2011, \"null\": 1}", "data_created_datetime": "2016-02-25T20:24:26Z", "courses_amount": 116}}, {"pk": 124, "model": "analytics.edxinstallation", "fields": {"latitude": 1.0, "platform_name": "test", "platform_url": "test", "longitude": 1.0, "secret_token": "platform 0"}}, {"pk": 125, "model": "analytics.installationstatistics", "fields": {"active_students_amount_day": 2925, "edx_installation": 125, "statistics_level": 1, "active_students_amount_month": 1000, "active_students_amount_week": 1000, "students_per_country": "{\"RU\": 2632, \"CA\": 18543, \"UA\": 2011, \"null\": 1}", "data_created_datetime": "2016-02-26T20:24:26Z", "courses_amount": 117}}, {"pk": 125, "model": "analytics.edxinstallation", "fields": {"latitude": 1.0, "platform_name": "test", "platform_url": "test", "longitude": 1.0, "secret_token": "platform 0"}}, {"pk": 126, "model": "analytics.installationstatistics", "fields": {"active_students_amount_day": 2950, "edx_installation": 126, "statistics_level": 1, "active_students_amount_month": 1000, "active_students_amount_week": 100
```

Comment:
Look at [utilities file](https://github.com/raccoongang/acceptor/blob/4e1f4c316cc9f06a8e8d90787b20f430b84a2386/edx_global_analytics/analytics/utils.py#L5). 
There is `Needed to be deleted in production.` string.
Make it multiline is unnecessary move. Especially for **test fixture**.